tests/src/main.rs

```diff
@@ -225,7 +225,7 @@ fn test_lane_changing(map: &Map) -> Result<()> {
     // This time limit was determined by watching the scenario manually. This test prevents the
     // time from regressing, which would probably indicate something breaking related to lane
     // selection.
-    let limit = Duration::minutes(8) + Duration::seconds(10.0);
+    let limit = Duration::minutes(8) + Duration::seconds(40.0);
```

Comment:
I watched this after and the overall behavior still seems reasonable, it just takes a little longer. All of our real maps continue to not gridlock; that's the bigger vote of confidence, in my mind.